docs/Mutators.md

```diff
@@ -164,9 +164,13 @@ Stryker supports a variety of mutators, which are listed below. Do you have a su
 | `abc{5}` | `abc` |
 | `abc{5,}` | `abc` |
 | `abc{5,8}` | `abc` |
+| `(?=abc)` | `(?!abc)` |
```

Comment:
The docs don't match with the implementation. If I'm correct `(?=foo)` mutates to `(?!foo)` and `(?<=foo)`.

I think the docs are correct but the implementation should change. It should only flip the positivity of the node and not the way it looks (ahead/behind).

Reply:
Yes good point, it was making too many mutants. Changed the code to only flip the negative/positive and leave look ahead and behind.